tests/python/gaia-ui-tests/gaiatest/apps/system/regions/search_panel.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The removal the rocketbar page class is not done here. I guess that's because we want to keep them for the testing the old homescreen, right?